### ISSUE TITLE
Last title is not tappable on right side

### ISSUE THREAD

**Issue by naqi** [HeshamMegid/HMSegmentedControl#146]
There seems to be a problem with the last title index. It is not tappable on its right side. Every where else works fine. Green is tappable zone and red is non-tappable. First found it on device and then reproduced in simulator. The indicator is tappable but was only able to make it work by precise cursor on simulator.

Any idea what could be the reason?

![screen shot 2015-10-07 at 12 28 51 am rectangle 1 rectangle 2](https://cloud.githubusercontent.com/assets/1274108/10329770/0f609706-6c8b-11e5-980f-90f8c74c9da1.png)

**Comment by naqi** [comment]
bump

**Comment by naqi** [comment]
TL;DR I was able to find a fix it by using IBOutlet rather than initializing it through code.

Opening the view hierarchy I was able to see that there were two instances of HMSegmentScrollView added to the view and somehow the first instance was taking over the touches for the red box in the picture. 

I am not sure how two instances were created though. I initialized and added it to view directory thinking it will give me more control. 

```
segment_control  = [[HMSegmentedControl alloc] initWithSectionTitles:@[@"People", @"Teams"]];

[self.view addSubView:segment_control];
```

Removed the above lines of code and using IBOutlet instance fixed the problem. Thoughts on what could have caused this?
